galvan/src/main/java/org/terracotta/testing/master/ServerProcess.java

```diff
@@ -212,24 +211,44 @@ private synchronized long waitForPid(){
     }
     return this.pid;
   }
-  
+
   private synchronized void reset(boolean running) {
     this.pid = 0;
     this.wasZapped = this.isRunning;
     this.isRunning = running;
     notifyAll();
   }
 
-  private String getStartScriptCommand() {
+  private String[] buildCommand(boolean consistentStart) {
     String startScript;
-    if (TestHelpers.isWindows()){
-      //There are illegal characters "(" and ")" in folder names that cause the path to be truncated and the server won't start.
-      //So we need to wrap double quotes around startScript absolute file path.
-      startScript = "\"" + new File(this.serverKitDirectory,"server\\bin\\start-tc-server.bat").getAbsolutePath() + "\"";
-    }else {
-      startScript = this.serverKitDirectory + "/server/bin/start-tc-server.sh";
+    String[] command;
+    if (configRepo != null) {
+      startScript = getStartNodeScript();
+      if (consistentStart) {
+        //use force flag as tc-config snippets may contain relative paths
+        command = new String[]{startScript, "-D", "-r", configRepo.toString(), "-f"};
```

Comment:
does start-node script supports "consistent startup"? It looks like we are starting in "diagnostic mode" here

Reply:
Thanks for raising it - I wanted to bring it up. @mathieucarbou is there an equivalent of the old script's "consistency mode" in the new script? If yes, I believe diagnostic mode is the  correct one?